### PR TITLE
feat: add storage of IPNS records in DynamoDB

### PR DESCRIPTION
**Pull request by guseggert**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -14,6 +15,7 @@ require (
 	github.com/go-kit/log v0.2.0
 	github.com/gopherjs/gopherjs v0.0.0-20190812055157-5d271430af9f // indirect
 	github.com/gorilla/mux v1.8.0
+	github.com/guseggert/go-ds-dynamodb v0.0.0-20220304162827-e7d473330642
```
Comment (guseggert) [inline go.mod:18]: Should I move this over to the IPFS org?

Comment (aschmahmann) [reply go.mod:18]: yep, moving it to ipfs sounds good. The alternative is ipfs-shipyard which is also a fine choice here.

**Review by aschmahmann — APPROVED** [review]
Looks right to me, have you tested this locally and seen things behaving as expected?

**Comment by guseggert** [comment]
Yes tested locally, and tested on a one-box in prod. It is currently running on all hosts in prod now.

**Pull request merged**